src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -108,5 +108,5 @@ public void getSalaryInfoByTwoMonths() {
                         + "\nexpected: \n" + expectedResult,
                 expectedResult,
                 actualResult);
-    }
+    }*/
```

Comment:
Sorry, but I cannot proceed with review when you commented out all the automated tests. Please restore default state for test class